### DISCUSSION TITLE
Cannot get blog to appear

### DISCUSSION THREAD

**Discussion by Drewmaloney** [nmelhado/league-page#295] [Help!]
Good afternoon, 
First, I'd like to thank you for this amazing work you have done. The site is incredible, the guys in my league love it, and I'm beyond appreciative you gave such an incredible tool like this. I'm still in the learning process to have a such an in depth template with data I'm familiar with has been a blessing, so thank you. 

I'm having an issue getting my blog posts to appear on my site. I searched the current discussions but couldn't find anyone else having issues. I followed your directions as closely as I could, the contentful UI has changed a little bit from your pictures but I think I did everything right but still had no success. Then, I connected my vercel to contentful through their UI so I know the tokens were created correctly and just changed the names to your naming conventions, but still have no luck getting the blog posts to appear. I'm sure I'm missing something obvious but not sure what. I've tried everything. Confirmed, that I updated enableblog to true.  

I have test blog posts in contentful. 
![Screenshot 2024-03-20 192412](https://github.com/nmelhado/league-page/assets/158063108/3714f02e-bed3-4a65-9553-6e4fd54e44ac)

Any direction would be appreciated. Would be happy to supply more info if I haven't supplied enough. 

Thank you, 
Drew

**Comment by scanale42** [comment]
Having the same problem. Created the environment variables enabled the blog, but nothing is posting.

**Comment by Drewmaloney** [comment]
I get these errors but not exactly sure how to investigate them. Haven't had much time lately though so I'll take a look and see what I can come up with. 
![image](https://github.com/nmelhado/league-page/assets/158063108/f2729c55-db48-4a7b-8c70-464d45550813)
![image](https://github.com/nmelhado/league-page/assets/158063108/668598b6-5ccc-4c05-9516-160ba774fa3e)

**Comment by taylordeal4** [comment]
I am also having this error. Followed the instructions to add the blog, but I cannot get it to show up on the actually league page.

**Comment by taylordeal4** [comment]
I am also having this issue.

**Comment by scanale42** [comment]
I got it working eventually, don’t recall what I had to do  I noticed there
were a bunch of items that were created that were in the instructions.  I
ended up deleting everything and recreating them and I think that fixed it.

**Comment by danielsutera** [comment]
Same issue here.

**Comment by dungeonsndankness** [comment]
I am now experiencing the same issue. Did anyone ever find a resolution to this? I followed all the instructions and can't get any data to come over.
